<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Manage Performance"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="57.743">
  <properties />
            <testcase name="a_test" time="57.743" classname="hcm.tests.ManagePerformanceRolesTest">
                <failure type="common.DuplicateNameEntryException"
                  message="Implementation Name already exists..
 Please select a unique Name for the Implementation Project"
              >
<![CDATA[common.DuplicateNameEntryException: Implementation Name already exists..
 Please select a unique Name for the Implementation Project
  hcm.tests.ManagePerformanceRolesTest.managePerf(ManagePerformanceRolesTest.java:85)
  hcm.tests.ManagePerformanceRolesTest.a_test(ManagePerformanceRolesTest.java:27)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:606)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:773)
  org.testng.TestRunner.run(TestRunner.java:623)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
  org.testng.SuiteRunner.run(SuiteRunner.java:259)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
  org.testng.TestNG.run(TestNG.java:1018)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
